test/e2e/apimachinery/flowcontrol.go

```diff
@@ -51,21 +51,26 @@ var _ = SIGDescribe("API priority and fairness", func() {
 		matchingUsername := "noxu"
 		nonMatchingUsername := "foo"
 
-		ginkgo.By("creating a testing prioritylevel")
+		ginkgo.By("creating a testing priority-level")
```

Comment:
I think we usually write simply "priority level", and either "flow schema" or "FlowSchema".